odoo/modules/module.py

```diff
@@ -432,12 +432,22 @@ def get_test_modules(module):
     modpath = 'odoo.addons.' + module
     try:
         mod = importlib.import_module('.tests', modpath)
+    except ImportError as e:  # will also catch subclass ModuleNotFoundError of P3.6
+        # Hide ImportErrors on `tests` sub-module, but display other exceptions
+        if pycompat.PY2:  # Python 2 backwards-compatibility
+            if e.message.startswith('No module named') and e.message.endswith("tests"):
+                pass
+            else:
+                _logger.exception('Can not `import %s`.', module)
```

Comment:
Maybe use `not` in the condition to avoid needing an empty branch? Likewise below.

Reply:
:+1:

Reply:
Done